atd-etl/cris_import/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN ln -snf /usr/share/zoneinfo/America/Chicago /etc/localtime && echo America/C
 RUN apt-get update && apt-get install -y tzdata
 
 RUN apt-get -y upgrade
-RUN apt-get install -y aptitude magic-wormhole vim black python3-pip
+RUN apt-get install -y aptitude magic-wormhole vim black python3-pip libmagic1 file
```

Comment:
We're using `libmagic` to check that the zips are zips through inspection of the binary file.